src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor();
+        Random random = new Random();
+        int number = random.nextInt(100);
```

Comment:
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();

    public Ball getRandomBall() {
        String color = colorSupplier.getRandomColor();
        int number = random.nextInt(100);
```
якщо ти оголошуєш змінну в методі, це означає, що вона буде створюватись нова під час кожного виклику методу. Так як нам не потрібна нова змінна рандому кожен раз для перформенсу буде краще якщо винесте це на ріаень класу